components/automate-chef-io/content/docs/backup.md

```diff
@@ -188,69 +188,81 @@ where `bucket_name` is the name of the S3 bucket `base_path` an optional path wi
-"backup.md#backup-to-filesystem" >}}):
+1. *(filesystem backups only)* Ensure access for Chef Automate to a backup directory in the [configured location]({{< ref
+"backup.md#backup-to-a-filesystem" >}}):
 
      1. To restore a network-attached filesystem backup, mount the shared backup directory to the same mount point configured at the time of the backup.
      1. To restore a backup directory that is not a network-attached filesystem, copy the backup directory to the location that was [configured]({{< ref
-"backup.md#backup-to-filesystem" >}}) at the time of the backup.
-     1. For restoring a single-file backup archive, copy your archive to the restore host and extract it to the [configured backup directory]({{< ref "backup.md#backup-to-filesystem" >}}).
+"backup.md#backup-to-a-filesystem" >}}) at the time of the backup.
+     1. For restoring a single-file backup archive, copy your archive to the restore host and extract it to the [configured backup directory]({{< ref "backup.md#backup-to-a-filesystem" >}}).
 
-#### Restore in an Internet-Connected Environment
+1. *(different FQDN only)* To restore to a host whose FQDN differs from that of the host on which the backup was
+   taken, create a `patch.toml` file to provide at restore time that specifies the new FQDN:
 
-If you have [configured the backup directory]({{< ref "backup.md#backup-to-filesystem" >}}) to a directory other than the default directory (`/var/opt/chef-automate/backups`) you must supply the backup directory. Without a backup ID, Chef Automate uses the most recent backup in the backup directory.
+    ```toml
+         [global.v1]
+         fqdn = "<new-fqdn>"
+
+         # To provide a cert and key for the restore host, uncomment and fill
+         # these sections.
+         # [[global.v1.frontend_tls]]
+         # The TLS certificate for the load balancer frontend.
+         # cert = """-----BEGIN CERTIFICATE-----
+         # <certificate-for-new-fqdn>
+         # -----END CERTIFICATE-----
+         # """
+
+         # The TLS RSA key for the load balancer frontend.
+         # key = """-----BEGIN RSA PRIVATE KEY-----
+         # <key-for-new-fqdn>
+         # -----END RSA PRIVATE KEY-----
+         # """
+    ```
 
-To restore on a new host:
+### Restore From a Filesystem Backup
 
-```shell
-chef-automate backup restore </path/to/backups/>BACKUP_ID
-```
+#### Restore in an Internet-Connected Environment
 
-To restore on an existing Chef Automate host:
+If you have [configured the backup directory]({{< relref "backup.md#backup-to-a-filesystem" >}}) to a directory other than the default directory (`/var/opt/chef-automate/backups`) you must supply the backup directory. Without a backup ID, Chef Automate uses the most recent backup in the backup directory.
 
 ```shell
-chef-automate backup restore </path/to/backups/>BACKUP_ID --skip-preflight
+chef-automate backup restore </path/to/backups/>BACKUP_ID [--patch-config </path/to/patch.toml>] [--skip-preflight]
```

Comment:
The plus of using the square brackets for non-required options is that the command and explanatory text can be more terse and less branchy. The minus is that it makes cutting and pasting a bit less convenient. Since cutting and pasting is already not immediate, I went for terseness.